packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -618,74 +425,20 @@ export class DeltaManager
             // (which in most cases will happen when we are done processing cached ops)
             if (cacheOnly) {
                 // fire and forget
-                this.fetchMissingDeltas("DocumentOpen", this.lastQueuedSequenceNumber);
+                this.fetchMissingDeltas("DocumentOpen");
             }
         }
 
         // Ensure there is no need to call this.processPendingOps() at the end of boot sequence
         assert(this.fetchReason !== undefined || this.pending.length === 0, 0x269 /* "pending ops are not dropped" */);
     }
 
-    private static detailsFromConnection(connection: IDocumentDeltaConnection): IConnectionDetails {
-        return {
-            claims: connection.claims,
-            clientId: connection.clientId,
-            existing: connection.existing,
-            checkpointSequenceNumber: connection.checkpointSequenceNumber,
-            get initialClients() { return connection.initialClients; },
-            maxMessageSize: connection.serviceConfiguration.maxMessageSize,
-            mode: connection.mode,
-            serviceConfiguration: connection.serviceConfiguration,
-            version: connection.version,
-        };
-    }
-
-    public async connect(args: IConnectionArgs): Promise<IConnectionDetails> {
-        const connection = await this.connectCore(args);
```

Comment:
This used to throw if connectCore hit a fatal error.  connectCore will still close container, but the new connect isn't async.  Do you know who's using this and how this will play out?

Reply:
Ok I see, it's just for logging everywhere. And I guess no one used the return value.  I see `IConnectionArgs` has `reason`, can we add that to errors thrown from connectCore as connectReason or something?

Reply:
Right, logging moved to ConnectionManager, now it's fire & forget. Given that reason is not used in ConnectionManager, I removed it from args. I'm not sure I want to add it just for logging - I have not seen a strong reason to have it - usually if we hit fatal error on connection, we are much more interested in why it failed, not why connection was initiated.

Reply:
Hmm ok. I don't have a strong counter-example in hand, so we can wait and see if we miss it :)